src/components/input/input.stories.ts

```diff
@@ -270,3 +270,9 @@ export const WithLabelAndInputMessageRTL = (): string => html`
     </calcite-label>
   </div>
 `;
+
+export const WithTibetNumberingSystem = (): string =>
+  html` <calcite-input type="number" numbering-system="tibt" value="123456"></calcite-input>`;
+
+export const ArabicLocaleWithLatinNumberingSystem = (): string =>
+  html` <calcite-input type="number" locale="ar" numbering-system="latn" value="123456"></calcite-input>`;
```

Comment:
I would use `ar-EG` (Egypt) here instead of just `ar` or one of the other ones that actually displays arabic numerals by default.  ☝️